samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/Data/wikipedia-detox-250-line-data.tsv

```diff
@@ -1,250 +0,0 @@
-Sentiment	SentimentText
```

Comment:
I see other references to this dataset:
https://github.com/dotnet/machinelearning-samples/search?q=wikipedia-detox-250-line-data.tsv&unscoped_q=wikipedia-detox-250-line-data.tsv

The reference may have their own copy.